assets/js/modules/adsense/components/dashboard/AdBlockingRecoveryWidget.test.js

```diff
@@ -205,6 +203,7 @@ describe( 'AdSenseConnectCTA', () => {
 		} );
 
 		it( 'should render the widget for the site with a setup completion time of more than three weeks', () => {
+			registry.dispatch( CORE_USER ).setReferenceDate( '2023-06-25' );
```

Comment:
This should not be needed and points to some newly introduced error (the dates being incorrect as mentioned above).

```suggestion
```